### ISSUE TITLE
Release Check failing on CI

### ISSUE THREAD

**Issue by jtpio** [jupyter-rtc/jupyterlab#40]
The `release_check` CI check is currently failing with:

```bash
> lerna publish from-package --no-git-tag-version --no-push  --dist-tag=next -m "Publish"
lerna notice cli v3.22.1
lerna info versioning independent
lerna info ci enabled
lerna WARN Yarn's registry proxy is broken, replacing with public npm registry
lerna WARN If you don't have an npm token, you should exit and run `npm login`
lerna WARN Unable to determine published version, assuming "@jupyterlab/docprovider" unpublished.
lerna WARN Unable to determine published version, assuming "@jupyterlab/nbmodel" unpublished.

Found 2 packages to publish:
 - @jupyterlab/docprovider => 3.1.0-alpha.3
 - @jupyterlab/nbmodel => 3.1.0-alpha.3

(node:2215) UnhandledPromiseRejectionWarning: Error: Command failed: lerna publish from-package --no-git-tag-version --no-push  --dist-tag=next -m "Publish"
    at checkExecSyncError (child_process.js:635:11)
    at Object.execSync (child_process.js:671:15)
    at Object.run (/home/runner/work/jupyterlab/jupyterlab/buildutils/lib/utils.js:230:43)
    at Command.<anonymous> (/home/runner/work/jupyterlab/jupyterlab/buildutils/lib/publish.js:58:15)
    at Command.listener [as _actionHandler] (/home/runner/work/jupyterlab/jupyterlab/buildutils/node_modules/commander/index.js:422:31)
    at Command._parseCommand (/home/runner/work/jupyterlab/jupyterlab/buildutils/node_modules/commander/index.js:975:14)
    at Command.parse (/home/runner/work/jupyterlab/jupyterlab/buildutils/node_modules/commander/index.js:778:10)
    at Object.<anonymous> (/home/runner/work/jupyterlab/jupyterlab/buildutils/lib/publish.js:77:21)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
(node:2215) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2215) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
? Are you sure you want to publish these packages? (ynH) Done in 168.18s.
```

Example run: https://github.com/jupyter-rtc/jupyterlab/runs/2259529359

Which might be related to these two new packages not being published on npm yet.

It would be fine to wait for the PR to be opened in JupyterLab before considering publishing them. This will also let folks provide feedback on the name of the packages if they have any.

**Comment by echarles** [comment]
> It would be fine to wait for the PR to be opened in JupyterLab before considering publishing them. This will also let folks provide feedback on the name of the packages if they have any.

+1

**Comment by dmonad** [comment]
Fixed by #53  - we will publish the packages before the release
